src/prefabs/pageWithProfile.tsx

```diff
@@ -458,6 +459,75 @@ const beforeCreate = ({
     return undefined;
   }
 
+  const profilePictureDisabledKinds = createBlacklist([
```

Comment:
Why enable Login token and google document here? Also, object property not supported yet